bin/create-config

```diff
@@ -23,6 +23,18 @@ function createConfig() {
         assetHost:       process.env.S3_ASSET_HOST_URL
       }
     }
+  } else if (!!process.env.WEBDAV_SERVER_URL){
```

Comment:
From https://github.com/bartt/ghost-webdav-storage-adapter#via-environment-variables it actually doesn't seem like this direct injection _isn't_ necessary. Just setting the environment variables is enough.

Can you 1. test for that, and if it works, 2. remove this?

Reply:
Yikes! This slipped out of my feed. Just a quick note; I'll be onto it within a day or two!

Reply:
That's true! I've tested it and removed the lines. I did leave a comment instead, though, so future devs don't get confused 🙂